frontend/src/MyApp.js

```diff
@@ -1,8 +1,15 @@
 import { Routes, Route } from "react-router-dom";
 import NavigationBar from "./components/navbar";
 import RecipePage from "./components/RecipePage";
+import Login from "./components/login";
+import Register from "./components/register";
+import Profile from "./components/profile";
```

Comment:
I know that most of these imports aren't yours, but I think we need more consistency on using `require()` instead of `import from`

Reply:
The whole frontend folder is using import from, so I followed that, but I did use require() in the backend part

Reply:
Pretty sure it's fine if we use import. Most of react code follows export default, import from ' '

https://create-react-app.dev/docs/importing-a-component/     React also encourages use of import statements

We only need to use require() in backend because we are consuming node.js modules. If you look at examples from the earlier project, this is also the format that is used. We just need consistent import in frontend and require in backend